packages/appstore/src/types/stores.types.ts

```diff
@@ -1,9 +1,21 @@
 import ConfigStore from 'Stores/config-store';
 import WalletStore from 'Stores/wallet-store';
+import { Authorize as TAuthorize } from '@deriv/api-types';
 
 export type TRootStore = {
     ui: Record<string, any>;
     client: Record<string, any>;
```

Comment:
@iman-fs We should type the data source itself so wont need to import types everywhere else and type variables again.

```js
import ConfigStore from 'Stores/config-store';
import WalletStore from 'Stores/wallet-store';
import { Authorize } from '@deriv/api-types';

type TAccountList = NonNullable<Authorize['account_list']>[number];

type TExtendedAccountList = TAccountList & {
    created_at: number;
};

type TClientStore = {
    // Todo: Use `Authorize['account_list']` once `@deriv/api-types` is updated.
    wallet_accounts: TExtendedAccountList[];
};

export type TRootStore = {
    ui: Record<string, any>;
    client: TClientStore;
    config: ConfigStore;
    wallet_store: WalletStore;
};
```

Then you can remove all the typings in `packages/appstore/src/modules/trading-hub/index.tsx`.